demo/src/routes/+layout.svelte

```diff
@@ -50,7 +50,7 @@
 </svelte:head>
 -->
 
-<div class="agnos-ui">
+<div class="agnos-ui" data-sveltekit-reload={$updated ? '' : 'off'}>
```

Comment:
I have one question, the only doc that I found about this property is here : https://kit.svelte.dev/docs/link-options#data-sveltekit-reload.

But it applies on a link, and there is no value associated to this attribute. Is there a more precise doc somewhere ?

Reply:
This is not super documented indeed.
I found this from a Rich Harris presentation + this comment:
https://github.com/sveltejs/kit/discussions/8665#discussioncomment-4760788

While the official doc includes this:
https://kit.svelte.dev/docs/configuration#version

Reply:
From my understanding, doing ```data-sveltekit-reload={$updated ? '' : 'off'}``` is doing the same thing as
```typescript
beforeNavigate(({willUnload, to}) => {
		if ($updated && !willUnload && to?.url) {
			// force reload of the page on navigation when a new version of the site has been detected
			location.href = to.url.href;
		}
	});
```
but in a more succinct way.
The important point was to include the ```pollInterval``` config to make sure the store ```updated``` is... updated :)